mix.exs

```diff
@@ -60,13 +60,14 @@ defmodule Xebow.MixProject do
 
   defp aliases do
     [
-      "compile.assets": "cmd npm run deploy --prefix ./assets",
+      "assets.compile": assets_compile(),
```

Comment:
I think this command needs to be `"cmd npm run deploy --prefix ./assets"` so that the assets compile every time `mix assets.compile` is invoked. So the check for compiled assets I think needs to happen in the `firmware` task.

Reply:
The private function runs `"cmd npm run deploy --prefix ./assets"` any time it's invoked, and just conditionally runs install.